SLHCUpgradeSimulations/Geometry/test/phase2_digi_reco_pixelntuple_cfg.py

```diff
@@ -111,8 +111,11 @@
 #process.FEVTDEBUGHLToutput_step = cms.EndPath(process.FEVTDEBUGHLToutput)
 
 # Schedule definition
-process.schedule = cms.Schedule(process.digitisation_step,process.L1simulation_step,process.L1TrackTrigger_step,process.digi2raw_step)
-process.schedule.extend(process.HLTSchedule)
```

Comment:
Do you not need HLTSchedule anymore?

Reply:
Not anymore. After #35858, `process.schedule` already holds the HLT Paths previously provided by `HLTSchedule` in configs such as this one (see the comment `# process.schedule imported from cff in HLTrigger.Configuration`) one line below.